test/SlowTests/Issues/RavenDB_22176.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Http;
+using Raven.Server.NotificationCenter.Notifications;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+using Sparrow.Server.Collections;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues;
+
+public class RavenDB_22176 : RavenDB_17068
+{
+    [RavenFact(RavenTestCategory.Indexes)]
+    public async Task GetCurrentNotificationsAsJson()
```

Comment:
Maybe something like `TestGetNotificationsEndpoint` would be more descriptive

Reply:
done